api/db/schema.rb

```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```

Comment:
Is this supposed to be here?

Reply:
No, it's not supposed to be here. Great catch!